tools/DynamoPackagesAnalyzer/DynamoPackagesAnalyzer.csproj

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.8.0" />
     <PackageReference Include="CsvHelper" Version="30.0.1" />
-    <PackageReference Include="Greg" Version="3.0.1.4707" />
+    <PackageReference Include="Greg" Version="3.0.2.6284" />
```

Comment:
Why are we changing the greg version only for this project?

Reply:
@reddyashish This project seems to be behind (not aligned with all the other project)